.env

```
@@ -72,6 +72,7 @@ NUMPY=latest
 PANDAS=latest
 PYTHON=3.8
 PYTHON_IMAGE_TAG=3.8
+PYTHON_ABI_TAG=cp38
```

```suggestion
PYTHON_ABI_TAG=cp39
```

I just merged the PR to drop Python 3.8, so can bump this new variable as well (and there will be some small merge conflicts to solve, sorry!)